plugin.video.formula1/resources/settings.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings version="1">
+    <section id="plugin.video.formula1">
+        <category id="general" label="30001">
+            <group id="1" label="30010">
+                <setting id="list.size" type="string" label="30011" help="">
+                    <level>0</level>
+                    <default>15</default>
+                    <constraints>
+                        <options>
+                            <option>5</option>
+                            <option>10</option>
+                            <option>15</option>
+                            <option>20</option>
+                            <option>25</option>
+                        </options>
+                    </constraints>
+                    <control type="spinner" format="string"/>
+                </setting>
+            </group>
+            <group id="2" label="30020">
+                <setting id="video.format" type="string" label="30021" help="">
+                    <level>0</level>
+                    <default>HLS (Adaptive)</default>
+                    <constraints>
+                        <options>
+                            <option>360p</option>
+                            <option>720p</option>
+                            <option>1080p</option>
+                            <option>HLS (Adaptive)</option>
+                        </options>
```

Comment:
Is it ok if I update https://kodi.wiki/view/Add-on_settings_conversion#type.3D.22labelenum.22 accordingly?

labelenum "values" do not need a `label`-argument in `option`-tags (in contrast to "lvalues").

Reply:
Ping @ronie

Reply:
thanx for the headsup. i've added a note to the example in the wiki.